src/js/components/current-page-menu-item.js

```diff
@@ -186,9 +187,9 @@ export function syncPage(page, force, priority) {
     showInfo(
       <span>
         <Icon name="close" />&nbsp;
-          {(error && error.message) || 'Failed syncing page!'}
+          {getIOTAStatusMessage(error) || 'Failed syncing page!'}
```

Comment:
This should happen in ledger-guard or in hw-app-iota, so that romeo.html can still just use the `error.message`
```JavaScript
try {
...
} catch (error) {
  // translate if TransporError
  if (error.statusCode) {
    error.message = getIOTAStatusMessage(error.statusCode);
  }
  throw error;
}
```